typescript/infra/config/environments/mainnet2/middleware/queries/verification.json

```diff
@@ -62,5 +62,6 @@
       "isProxy": false,
       "constructorArguments": "0x"
     }
-  ]
+  ],
+  "gnosis": []
```

Comment:
Did you leave this empty intentionally? I see you mentioned manual verification in the PR desc

Reply:
yeah that's just what the deploy script ends up doing, it's not material really and will be wiped whenever @yorhodes does the next full deploy